CPurple/metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+    "last-edited": 1703441316
+}
```

Comment:
This file should not be here. Also, you might consider adding a README file. 

This is also missing two key files:
1. You should have a screenshot.png displaying the theme with icons, etc. (Check the Spices site for others.)
2. You should have an info.json with the following contents:
```json
{
    "author": "TheOnlyJoeEnderman"
}
```

Reply:
all issues listed plus license have been fixed.